ui/src/app/views/workspace-share/component.css

```diff
@@ -82,7 +79,6 @@ clr-dropdown > button {
 }
 
 .paragraph-text {
```

Comment:
Could you put some padding or margin under this? The divider looks uncomfortably close to this text to me.

Reply:
Will do. Agreed.